menu.go

```diff
@@ -3,24 +3,28 @@ package fyne
 // Menu stores the information required for a standard menu.
 // A menu can pop down from a MainMenu or could be a pop out menu.
 type Menu struct {
-	Label string
-	Items []*MenuItem
+	Label            string
+	Items            []*MenuItem
+	AfterNativeMenus bool
 }
 
 // NewMenu creates a new menu given the specified label (to show in a MainMenu) and list of items to display.
 func NewMenu(label string, items ...*MenuItem) *Menu {
-	return &Menu{label, items}
+	return &Menu{Label: label, Items: items}
 }
 
-// MenuItem is a sligne item within any menu, it contains a dispay Label and Action function that is called when tapped.
+// MenuItem is a single item within any menu, it contains a display Label and Action function that is called when tapped.
 type MenuItem struct {
-	Label  string
-	Action func()
+	Label             string
+	KeyEquivalent     string
+	PlaceInNativeMenu bool
+	Separate          bool
```

Comment:
This approach is different from the rest of the Fyne API in which separators are implemented as independent objects, eg "ToolbarSeparator". How about defining "MenuItemSeparator", or "MenuSeparator" instead?

Reply:
Yes, you are right.